### ISSUE TITLE
Cannot stream to Facebook Live

### ISSUE THREAD

**Issue by benshawuk** [carlosonunez/obs-installer-for-apple-silicon#21]
Trying to stream doesn't connect - it instantly fails with an error - "Failed to connect to server" as though it isn't attempting connection at all.

**Comment by carlosonunez** [comment]
I actually haven't used OBS for streaming, but can you try enabling logging and sending them to me? https://obsproject.com/tools/analyzer?log_url=

**Comment by carlosonunez** [comment]
Might be a missing build flag. I'll look into this. While I do that, can you send me the full log (with any credentials scrubbed)?

**Comment by allesgroupadmin** [comment]
> Might be a missing build flag. I'll look into this. While I do that, can you send me the full log (with any credentials scrubbed)?
> […](#)

Absolutely, if there's anything you can think of that you'd like me to try & you're busy, this is a problem for me so I can spend some time trying to fix this lol. https://obsproject.com/logs/MJaati6v8GURyIYa

**Comment by carlosonunez** [comment]
I know this isn't what you're experiencing, but does OBS normally have decklink support enabled?

**Comment by carlosonunez** [comment]
Good to know. I'll look into it.

**Comment by gp-Airee** [comment]
> @allesgroupadmin I just pushed a new release. Can you give it a try?

Absolutely, thank you! I'll get back after testing

**Comment by allesgroupadmin** [comment]
Absolutely, I'll let you know how it goes.

-------- Original Message --------

**Comment by benshawuk** [comment]
Hi @carlosonunez - many thanks for looking at this issue!  (And thanks @allesgroupadmin too!)
I'm also unable to create the build successfully.  Here is my build log:
https://0bin.net/paste/74ap0MYE#MZ9kTSRRp-D8eAvrYOy62n4ikabSC7T4nU17Ly+fNhV

I'm also happy to carry out any testing etc. to help get this functionality working.

**Comment by carlosonunez** [comment]
This link 404s for me; can you try again?

**Comment by allesgroupadmin** [comment]
> > Build failed: https://0bin.net/paste/JrJu3HqQ#O3gxggwsnIHIG-FLba9Z2/9oU8rEm60W+jLd49Gp6PE
> 
> @allesgroupadmin can you send me a different link? This 404s.

Absolutely, here you go: https://0bin.net/paste/a3XTpSvk#t3VTrEqa1J+JS8wjv2AePMMMGrGgNX71Nvh15DoCWdE

**Comment by carlosonunez** [comment]
> Hi @carlosonunez - many thanks for looking at this issue! (And thanks @allesgroupadmin too!) I'm also unable to create the build successfully. Here is my build log: https://0bin.net/paste/74ap0MYE#MZ9kTSRRp-D8eAvrYOy62n4ikabSC7T4nU17Ly+fNhV
> 
> I'm also happy to carry out any testing etc. to help get this functionality working.

@benshawuk It's interesting that you're getting this, since these scripts don't set a download strategy. Can you paste the output from this command and report back? (Be mindful of passwords.)

```sh
env | grep -i homebrew
```

**Comment by carlosonunez** [comment]
> > > Build failed: https://0bin.net/paste/JrJu3HqQ#O3gxggwsnIHIG-FLba9Z2/9oU8rEm60W+jLd49Gp6PE
> > 
> > 
> > @allesgroupadmin can you send me a different link? This 404s.
> 
> Absolutely, here you go: https://0bin.net/paste/a3XTpSvk#t3VTrEqa1J+JS8wjv2AePMMMGrGgNX71Nvh15DoCWdE

@allesgroupadmin Interesting, as this was definitely fixed in my most recent version.

Can you run this command again, but with `LOG_LEVEL=verbose` prepended to it? Please delete `/tmp/obs`, `/tmp/vlc-obs` and `/tmp/cef-obs` before doing that. Thanks!

**Comment by benshawuk** [comment]
> > Hi @carlosonunez - many thanks for looking at this issue! (And thanks @allesgroupadmin too!) I'm also unable to create the build successfully. Here is my build log: https://0bin.net/paste/74ap0MYE#MZ9kTSRRp-D8eAvrYOy62n4ikabSC7T4nU17Ly+fNhV
> > I'm also happy to carry out any testing etc. to help get this functionality working.
> 
> @benshawuk It's interesting that you're getting this, since these scripts don't set a download strategy. Can you paste the output from this command and report back? (Be mindful of passwords.)
> 
> ```shell
> env | grep -i homebrew
> ```

Hi @carlosonunez , sure.  Here is the output:

MANPATH=/opt/homebrew/share/man:
HOMEBREW_REPOSITORY=/opt/homebrew
PATH=/opt/homebrew/bin:/opt/homebrew/sbin:/usr/local/opt/php@7.4/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Library/Apple/usr/bin
HOMEBREW_PREFIX=/opt/homebrew
INFOPATH=/opt/homebrew/share/info:
HOMEBREW_CELLAR=/opt/homebrew/Cellar

**Comment by allesgroupadmin** [comment]
@carlosonunez Any news?

**Comment by carlosonunez** [comment]
Merging this with #25 ; @benshawuk , please re-raise if you're still having issues connecting to Facebook.
